layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -645,11 +645,13 @@ variable."
   (ediff-files (dotspacemacs/location)
                (concat dotspacemacs-template-directory ".spacemacs.template")))
 
+(defvar spacemacs-new-empty-buffer-major-mode nil "Set the major mode for a new empty buffer.")
```

Comment:
Formatting suggestion:
The other variables in the same file, shows the docstring on the line below the variable definition. This also keeps the line under 80 chars. And they leave an empty line between the variable and the following function.

For example:
https://github.com/syl20bnr/spacemacs/blob/ad5dfc1ad5c3d08266141487397a83810fa7fa7f/layers/%2Bspacemacs/spacemacs-defaults/funcs.el#L910-L913

Reply:
Formatting suggestion applied.